sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -636,6 +636,22 @@ def _add_argparse_args(cls, parser):
          'same order they were specified on the command line.'))
 
 
+class PortableOptions(PipelineOptions):
+
+  @classmethod
+  def _add_argparse_args(cls, parser):
+    parser.add_argument('--job_endpoint',
+                        default=None,
+                        help=
+                        ('Job service endpoint to use. Should be in the form '
+                         'of address and port, e.g. localhost:3000'))
+    parser.add_argument('--docker_image',
```

Comment:
Shall we use --worker_harenss_image?

Referece:
https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java#L123

Reply:
I considered it, but it is the image of the Dataflow *runner* harness, not the SDK harness.

Reply:
I would still like to make it explicit form the name that the image is for SDKHarness means it needs to support FnApi.